### ISSUE TITLE
Error while build netlify cms plugin

### ISSUE THREAD

**Issue by Jaikant** [gatsbyjs/gatsby#4071]
I am getting the below error while building the netlify cms plugin. 

This was working a couple of weeks back. Not sure what went wrong.

@erquhart could you help / point me in the right direction?

 ```
Error: ./~/netlify-cms/dist/cms.js
  Module parse failed: /Users/jai/work/guides/consulting/mountainadvice/gatsby-theme-kit/no  de_modules/netlify-cms/dist/cms.js Unexpected token (174:70542)
  You may need an appropriate loader to handle this file type.
  SyntaxError: Unexpected token (174:70542)
      at Parser.pp$4.raise (/Users/jai/work/guides/consulting/mountainadvice/gatsby-theme-k  it/node_modules/acorn/dist/acorn.js:2221:15)
      at Parser.pp.unexpected (/Users/jai/work/guides/consulting/mountainadvice/gatsby-them  e-kit/node_modules/acorn/dist/acorn.js:603:10)
      at Parser.pp.semicolon (/Users/jai/work/guides/consulting/mountainadvice/gatsby-theme  -kit/node_modules/acorn/dist/acorn.js:581:61)
      at Parser.pp$1.parseReturnStatement (/Users/jai/work/guides/consulting/mountainadvice  /gatsby-theme-kit/node_modules/acorn/dist/acorn.js:839:57)
      at Parser.pp$1.parseStatement (/Users/jai/work/guides/consulting/mountainadvice/gatsb  y-theme-kit/node_modules/acorn/dist/acorn.js:699:34)
      at Parser.pp$1.parseBlock (/Users/jai/work/guides/consulting/mountainadvice/gatsby-th  eme-kit/node_modules/acorn/dist/acorn.js:981:25)
      at Parser.pp$3.parseFunctionBody (/Users/jai/work/guides/consulting/mountainadvice/ga  tsby-theme-kit/node_modules/acorn/dist/acorn.js:2105:24)
      at Parser.pp$1.parseFunction (/Users/jai/work/guides/consulting/mountainadvice/gatsby  -theme-kit/node_modules/acorn/dist/acorn.js:1065:10)
      at Parser.pp$1.parseFunctionStatement (/Users/jai/work/guides/consulting/mountainadvi  ce/gatsby-theme-kit/node_modules/acorn/dist/acorn.js:818:17)
      at Parser.pp$1.parseStatement (/Users/jai/work/guides/consulting/mountainadvice/gatsb  y-theme-kit/node_modules/acorn/dist/acorn.js:694:19)
      at Parser.pp$1.parseBlock (/Users/jai/work/guides/consulting/mountainadvice/gatsby-th  eme-kit/node_modules/acorn/dist/acorn.js:981:25)
      at Parser.pp$3.parseFunctionBody (/Users/jai/work/guides/consulting/mountainadvice/ga  tsby-theme-kit/node_modules/acorn/dist/acorn.js:2105:24)
      at Parser.pp$1.parseFunction (/Users/jai/work/guides/consulting/mountainadvice/gatsby  -theme-kit/node_modules/acorn/dist/acorn.js:1065:10)
      at Parser.pp$3.parseExprAtom (/Users/jai/work/guides/consulting/mountainadvice/gatsby  -theme-kit/node_modules/acorn/dist/acorn.js:1810:19)
      at Parser.pp$3.parseExprSubscripts (/Users/jai/work/guides/consulting/mountainadvice/  gatsby-theme-kit/node_modules/acorn/dist/acorn.js:1715:21)
      at Parser.pp$3.parseMaybeUnary (/Users/jai/work/guides/consulting/mountainadvice/gats  by-theme-kit/node_modules/acorn/dist/acorn.js:1692:19)
      at Parser.pp$3.parseExprOps (/Users/jai/work/guides/consulting/mountainadvice/gatsby-  theme-kit/node_modules/acorn/dist/acorn.js:1637:21)
      at Parser.pp$3.parseMaybeConditional (/Users/jai/work/guides/consulting/mountainadvic  e/gatsby-theme-kit/node_modules/acorn/dist/acorn.js:1620:21)
      at Parser.pp$3.parseMaybeAssign (/Users/jai/work/guides/consulting/mountainadvice/gat  sby-theme-kit/node_modules/acorn/dist/acorn.js:1597:21)
      at Parser.pp$3.parseExprList (/Users/jai/work/guides/consulting/mountainadvice/gatsby  -theme-kit/node_modules/acorn/dist/acorn.js:2165:22)
      at Parser.pp$3.parseExprAtom (/Users/jai/work/guides/consulting/mountainadvice/gatsby  -theme-kit/node_modules/acorn/dist/acorn.js:1801:28)
      at Parser.pp$3.parseExprSubscripts (/Users/jai/work/guides/consulting/mountainadvice/  gatsby-theme-kit/node_modules/acorn/dist/acorn.js:1715:21)
   @ ./src/cms/cms.js 7:18-40
```

**Comment by m-allanson** [comment]
@Jaikant there's a pending fix for this, see #4046.

**Comment by Jaikant** [comment]
Closing as it is a duplicate! Sorry about that. 

https://github.com/gatsbyjs/gatsby/issues/4031
